client/src/components/Cell/CellSkills.tsx

```diff
@@ -17,6 +18,15 @@ const CellSkills = ({
     [brute, randomSkillIndex]
   );
 
+  const getFilter = useMemo(
+    () => (skill: SkillName) => {
+      if (randomSkill === skill) return 'drop-shadow(0 0 0.5rem #ff0000)';
+      if (brute?.ascendedSkills.includes(skill)) return 'drop-shadow(0 0 0.5rem #ff9400)';
+      return 'none';
+    },
+    [randomSkill, brute]
+  );
```

Comment:
You shouldn't memo functions, as they will never be properly cached, a simple `getFilter = (skill: SkillName) => ...` is enough